### ISSUE TITLE
ctsm5.3.032: Add CRU-JRA as option for datm inputs for CTSM development

### ISSUE THREAD

**Issue by wwieder** [ESCOMP/CTSM#1895]
Should we consider updating to CRUJRA as our default forcing for CTSM development?

**Brief +/- here:** 
- Positive here are that these are the input data used for TRENDY simulations and updated annually (and currently available through 2021, vs. GSWP3, which ends in 2014).
- Negatives are that each year it seems like the dataset is slightly modified. We may not what to update our input data this often. 

**Considerations:**
- We need some assessment of what results look like with these simulations, but @djk2120 has CLM50 runs for TRENDY;
- We should update to CTSM5.1 / 5.2 branch and spin up new initial conditions, which would be helpful to discuss with @olyson and the rest of the LMWG/CLM team.
- This also will require developing some infrastructure / storage to support a new input dataset that @ekluzek can inform.
- @swensosc also noted CRU-JRA is 6 hourly data, whereas GSWP3 is 3 hr resolution.
- @olyson noted there's not data for Antarctica with CRU-JRA, complicating evaluation.

**There have been a number of conversations on this related to:**
- TRENDY simulations #1701,
- Roughness calculation #1596, and
- Mesh creation #1677

We can have discussion on this thread, but I also suggest we discuss at an upcoming Thursday science meeting (maybe next week)?

---
Updated Nov 20, 2024

In the absence of other options of datm data to bring us through present day, we're calibrating CLM6 with CRU-JRA forcing.  A few notes related to outstanding tasks

**Definition of done**
- [x] Fill data over Antarctica with GSPW3-forcing
  - Data available at `/glade/derecho/scratch/wwieder/TRENDY2024/inputs/three_stream`  
- [x] #2989
- [x] In same issue #2989 discuss sharing @adrifoster and @swensosc's script that takes raw CRUJRA data to make three stream datm files. Initial thoughts from @wwieder and @slevis-lmwg that this can go in /tools/contrib/.
- [x] load CRUJRA datm and mesh files appropriately onto input data (also a check box in #2675).
- [x] @slevis-lmwg set up CLM5 and CLM6 compset so that users get datm.streams set up correctly out of the box
  - [x] Clm6 and include necessary mods to cdeps (open PR there, too) and any other submodules
  - [x] Clm5 (repeat whatever's needed)
- [x] @slevis-lmwg `./rimport -list crujra2024.list` (rerunning rimport_slevis with the newly recommended path)
- [x] #2960
  -  @wwieder has done this in a hacky way, just replacing GSWP3 with CRUJRA in the same brittle way on the mimics_AAtest_ctsm5.2.028 branch I have on derecho.

**Comment by wwieder** [comment]
@djk2120 noted that the forcing is available here:
/glade/campaign/asp/djk2120/TRENDY2022/forcing

**Comment by djk2120** [comment]
latest TRENDY from GCB-2021:
/glade/campaign/asp/djk2120/TRENDY2022/S3/lnd

based on:
branch_tags/TRENDY-2019.n05_ctsm1.0.dev056

**Comment by wwieder** [comment]
@olyson can you run diagnostics (and ILAMB) on these results?  Does it make the most sense to compare them with results from the CLM5.0-GWSP3 runs that were done for the CLM5 release?

**Comment by olyson** [comment]
Either that or the more recent CLM5 simulation I did with the PPE tag: cesm2_3_alpha02c_PPE.n08_**ctsm5.1.dev030**

for our comparison with "CTSM5.1":

https://github.com/ESCOMP/CTSM/wiki/CLM5-CTSM5.1-Simulations

**Comment by olyson** [comment]
Either way, I'll need permissions set on Daniel's directory to at least be able to read/copy the files on campaign store.  Currently they are set to "asp" group which I don't have permissions for.  

drwxr-s---+ 11 djk2120 asp      4096 Oct 12 11:33 TRENDY2022

**Comment by wwieder** [comment]
Yep the CTSM5.1.dev030 tag is also a good candidate, Keith.  Dealers choice.  Again, our main goal here will be to start learning big difference in the forcing data and what differences we may expect at initialization and in historical trajectories.  Down the road we'll do a clean run with a modern tag and both forcing datasets assuming there's no obvious flaws in CRU-JRA

**Comment by wwieder** [comment]
I've learned that JRA is no longer going to be continue / supported, which makes me think that CRU-JRA, as used in TRENDY will be of limited utility moving forward.  The need for updated datm data is going to be a need for multiple CESM communities (ocean and atm use JRA for initialization in the ESPWG).  At this point I suggest we wait to see what the TRENDY migrates to or consider a larger effort to generate these input data. @billsacks mentioned that Kevin R in DART suggested they may be a data product that could serve this need.

**Comment by klindsay28** [comment]
FYI, with the upcoming stoppage in JRA support, the ocean community is also faced with deciding how to move forward. My understanding, based on conversations w/ Gokhan Danabasoglu, is that the ocean community is leaning towards using ERA5 from ECMWF. The forcing spans 1940 to present day, and ECMWF seems committed to updating it. I don't know if this forcing meets the needs for CTSM/CLM. It seems prudent for conversations within CESM on moving forward to include folks from both the LMWG and OMWG, to ensure coordination, where it is possible.

**Comment by dlawrenncar** [comment]
I was going to suggest the same thing.  It would be great if we could
create something that would be applicable for both ocean and land groups.
1940 is not as early as we would like, but I wonder if that limitation
might be offset by having a routinely updatable dataset to use.

**Comment by swensosc** [comment]
ecmwf also has a few products that go back to 1900:
https://www.ecmwf.int/en/forecasts/datasets/browse-reanalysis-datasets
It seems like we could splice together one of those with the newer updated
data.  Or maybe dart could be used to assimilate one into the other to
synthesize them.

**Comment by wwieder** [comment]
@adrifoster created CRUJRAv2.5 data for use in the TRENDY 2024 simulations. 
`/glade/campaign/cgd/tss/projects/TRENDY2024/inputs/three_stream/`

These annual files, 1901-2023, are at 0.5 degree, 6 hr resolution but are still missing data over Antarctica.  This is OK for running I cases, but it is suboptimal for generating initial conditions for use in B and F cases (#2403). 
 
I updated these files pasting in data south of 60S from GSWP3 (also 0.5 resolution, but every 3 hours).  For now I've only made 20 years of data (1901-1920).  New data are here `/glade/derecho/scratch/wwieder/TRENDY2024/inputs/three_stream/`, and I'll also put them on Izumi
`/scratch/cluster/wwieder/TRENDY2024`

Code is on [github](https://github.com/wwieder/ctsm_py/blob/master/notebooks/CRUJRA_antarctica.ipynb). 
At some point it would be helpful for someone to have a look at the script and files to see if this is working correctly.  We can also start generating new 5.3 initial conditions files (with Li2024 fire turned on with the f09 and ne30 grids).

**Comment by slevis-lmwg** [comment]
Also we should soon (before /glade's scrubber gets us) move the datm files and corresponding mesh file to /glade/campaign/.../atm/datm7/...

I'm adding a checkbox in issue #2675.

**Comment by slevis-lmwg** [comment]
In the context of 
https://github.com/NCAR/cesm_dev/discussions/26 and https://github.com/NCAR/LMWG_dev/discussions/75
@wwieder @linniahawkins @dlawrenncar @slevis-lmwg 
discussed (on 2024/12/16) whether it still makes sense to make CRU-JRA the default due to its low humidity values.

**Comment by slevis-lmwg** [comment]
Note to self: Using `git grep -i gswp` to find all the references to the current default.

**Comment by slevis-lmwg** [comment]
**Seeking feedback**
I will name the new default "CRUJRA" but do we want a version associated with the name? In the past we have included version numbers, such as CRUv7 and GSWP3v1. For the new default I see two version label options:
- v2.5 (listed on the files)
- 2024 (listed on the atm_forcing directory)

**Comment by ekluzek** [comment]
@slevis-lmwg I think we should call it CRUJRA2024 for the long name. You could add dashes or periods or something in there to make it easier to read.

**Comment by wwieder** [comment]
I agree we need something for the version of this dataset.  @ekluzek's suggestion makes sense, especially if it's effectively a new dataset for each TRENDY edition.  Do they ever just append the version number with an additional year? Maybe @adrifoster or @djk2120 can weigh in here?

**Comment by wwieder** [comment]
This PR is waiting on cdeps tag from @jedwards4b and another review from @ekluzek
